master/buildbot/test/unit/test_www_hooks_bitbucket.py

```diff
@@ -14,124 +14,51 @@
 # Copyright Buildbot Team Members
 # Copyright Manba Team
 import calendar
+from StringIO import StringIO
+
+from dateutil.parser import parse as dateparse
 
 from twisted.internet.defer import inlineCallbacks
 from twisted.trial import unittest
 
 import buildbot.www.change_hook as change_hook
 from buildbot.test.fake.web import FakeRequest
 from buildbot.test.fake.web import fakeMasterForHooks
+from buildbot.www.hooks.bitbucket import _HEADER_CT
+from buildbot.www.hooks.bitbucket import _HEADER_EVENT
+from www_hooks_bitbucket_json_data import gitJsonNoCommitsPayload
```

Comment:
i wonder why these imports are not flagged by our system

a) they should fully qualified path
b) order